QSB/TransformSync/NomaiOrbTransformSync.cs

```diff
@@ -0,0 +1,89 @@
+﻿using QSB.WorldSync;
+using UnityEngine;
+using UnityEngine.Networking;
+
+namespace QSB.TransformSync
+{
+    public class NomaiOrbTransformSync : NetworkBehaviour
```

Comment:
why not inherit from TransformSync?

Reply:
don't need the sector sync

Reply:
TransformSync's variables and whole structure is based on sectors... for something to use that when it doesnt use sectors seems wrong